Userland/Libraries/LibWeb/UIEvents/WheelEvent.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/TypeCasts.h>
```

Comment:
TypeCasts isn't used in the header, if it's used in the cpp it should be included there instead (or just removed)